### DISCUSSION TITLE
Return the old data storage format

### DISCUSSION THREAD

**Discussion by JerryLutor** [minio/minio#15238] [Q&A] [answered]
How to return the old data storage format ?

In the new version, the format has changed to: "format":"xl-single" .

how to return files to be files and not a bunch of folders with xl.meta and part.1 ?

**Comment by harshavardhana** [comment, marked as answer]
@JerryLutor if you have existing data it will work as is. Fresh deployments are not supported with legacy format anymore.

Please read #14331

**Reply by JerryLutor** [reply]
Thanks
